app/models/transformation_mapping.rb

```diff
@@ -6,4 +6,72 @@ class TransformationMapping < ApplicationRecord
   def destination(source)
     transformation_mapping_items.find_by(:source => source).try(:destination)
   end
+
+  def validate_vms(csv_list = nil)
+    csv_list.present? ? identify_vms(csv_list) : select_vms
+  end
+
+  private
+
+  def select_vms
+    # TBD
+  end
```

Comment:
Why add a method if you aren't going to use it...just add it later when you need it.

Reply:
It is an sample for our team developer. It will be filled with logic very soon. It is urgently needed.

Reply:
I think @Fryguy is saying anyone can add the ternary above when there is a usage for it.

Reply:
@bzwei Can you confirm that @lfu is already working on the followup PR to this?  This is not for some possible future usage, it will be used imminently.

Reply:
@gmcculloug The follow up PR is coming today. Working on the test cases.